tfgrid3/ubuntu22.04/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:22.04
+RUN echo deb http://be.archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse > /etc/apt/sources.list
+RUN apt update && \
+  apt -y install wget curl vim net-tools iputils-ping openssh-client openssh-server docker.io
```

Comment:
please install ssh only, no need for the other packages

Reply:
done.